src/configs/store/auth/actions.js

```diff
@@ -0,0 +1 @@
+export const setAuth = 'AUTH/SET_AUTH';
```

Comment:
Why are you using 'AUTH' in variable value. I think 'SET_AUTH' will be enough.

Reply:
In application can be a lot of actions, it is better to write "FEATURE/NAME_OF_ACTION"

Reply:
for example https://www.npmjs.com/package/typesafe-actions

Reply:
I don't have any problem here